### DISCUSSION TITLE
Welcome to apiato Discussions!

### DISCUSSION THREAD

**Discussion by Mohammad-Alavi** [apiato/apiato#707] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inuryyev** [comment]
Hi. I work as a full-stack developer, and I developed an API using the Apiato framework for a mobile app.
Personally, I think this framework is great and I believe it should become more popular.
